wallbox_client.py

```diff
@@ -419,10 +427,8 @@ def process_soc(self, reported_soc: str) -> bool:
 
         # Notify user of reaching 80% charge while charging (not dis-charging).
         # ToDo: Discuss with users if this is useful.
-        # ToDo: Replace 80 with max soc setting from yaml.
-        notify_soc_setting = 80
-        if self.connected_car_soc == notify_soc_setting and self.is_charging():
-            self.notify_user(f"Car battery state of charge has reached {notify_soc_setting}%.")
+        if self.connected_car_soc == c.CAR_MAX_SOC_IN_PERCENT and self.is_charging():
+            self.notify_user(f"Car battery at {self.connected_car_soc}%, range ≈ {tmp}km.", None, "battery_max_soc_reached", 60*15)
```

Comment:
add space before km

Reply:
According to international standards (for technical documentation) there should be a space between number and UoM. To make sure they are not separated over two lines a non-breaking narrow " " space is used.